### PR TITLE
Run build on deployment host

### PR DESCRIPTION
**Pull request by moshthepitt**

### CODE REVIEW

**Comment by p-netm** [comment]
@moshthepitt please help disable drone integration, I tried doing so, but it only shows a toast saying the operation failed.

**Comment by moshthepitt** [comment]
@p-netm should be done.  Also added you as an admin to this repo

**Comment by p-netm** [comment]
> @p-netm should be done. Also added you as an admin to this repo

Thanks.

**Comment by p-netm** [comment]
@moshthepitt this is good to go.

File: .yamllint [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  hyphens:
+    level: error
+  indentation: disable
+  key-duplicates: enable
+  line-length: disable
+  new-line-at-end-of-file: disable
+  new-lines:
+    type: unix
+  trailing-spaces: disable
+  truthy: disable
```
Comment (morrismukiri) [inline .yamllint:33]: Might want to remove these

**Review by ukanga — APPROVED** [review]

**Pull request merged**